src/mcts/stoppers/alphazero.cc

```diff
@@ -83,7 +80,7 @@ std::unique_ptr<SearchStopper> AlphazeroTimeManager::GetStopper(
   // Make sure we don't exceed current time limit with what we calculated.
   auto deadline =
       std::min(static_cast<int64_t>(this_move_time), *time + increment - move_overhead_);
```

Comment:
With the change suggested above, this is not needed (and increment shouldn't be added).

Reply:
Have created a sheet with some test values, see other comment